### PR TITLE
Release 27.10

### PR DESCRIPTION
**Pull request by lucasmoura**

### CODE REVIEW

File: docs/howtoguides/configure_proxies.md [hunk 1]
```diff
@@ -1,17 +1,100 @@
...
+> **Note**
+> Any configuration file that comes later in the apt.conf.d
+> directory could override the proxy configured by the UA client.
+
+To remove the APT proxy configuration, run the following:
+
+$ sudo ua config unset apt\_http\_proxy
+$ sudo ua config unset apt\_https\_proxy
+
+> **Note**
```
Comment (CalvoM) [inline docs/howtoguides/configure_proxies.md:61]: Considering this is 27.10, could we add that `apt_http(s)_proxy` will be deprecated.

Comment (lucasmoura) [reply docs/howtoguides/configure_proxies.md:61]: Yes, I think we can improve on that, but let's use a separate PR for this. The main goal of this PR is just to align on what we will release next and not to provide any extra functionality

**Review by renanrodrigo — APPROVED** [review]
LGTM
ok to wait a little if we want to include the other doc PR about the deprecation of the proxy vars

**Comment by lucasmoura** [comment]
@renanrodrigo I don't think this is necessary. We don't ship the docs in the package, so we can do this in parallel

File: docs/howtoguides/configure_proxies.md (hunk 1 above)
Comment (CalvoM) [reply docs/howtoguides/configure_proxies.md:61]: Okay. No problem.

**Pull request merged**